src/components/AvatarWithImagePicker.js

```diff
@@ -252,41 +241,23 @@ class AvatarWithImagePicker extends React.Component {
-                                                    <Tooltip absolute text={this.props.translate('avatarWithImagePicker.editImage')}>
-                                                        <View style={[styles.smallEditIcon, styles.smallAvatarEditIcon]}>
-                                                            <Icon
-                                                                src={Expensicons.Camera}
-                                                                width={variables.iconSizeSmall}
-                                                                height={variables.iconSizeSmall}
-                                                                fill={themeColors.iconReversed}
-                                                            />
-                                                        </View>
-                                                    </Tooltip>
-                                                    <PopoverMenu
-                                                        isVisible={this.state.isMenuVisible}
-                                                        onClose={() => this.setState({isMenuVisible: false})}
-                                                        onItemSelected={() => this.setState({isMenuVisible: false})}
-                                                        menuItems={this.createMenuItems(openPicker)}
-                                                        anchorPosition={this.props.anchorPosition}
-                                                    />
-                                                </>
-                                            )
-                                    }
+                                    <Tooltip absolute text={this.props.translate('avatarWithImagePicker.editImage')}>
+                                        <View style={[styles.smallEditIcon, styles.smallAvatarEditIcon]}>
+                                            <Icon
+                                                src={Expensicons.Camera}
+                                                width={variables.iconSizeSmall}
+                                                height={variables.iconSizeSmall}
+                                                fill={themeColors.iconReversed}
+                                            />
+                                        </View>
+                                    </Tooltip>
+                                    <PopoverMenu
+                                        isVisible={this.state.isMenuVisible}
+                                        onClose={() => this.setState({isMenuVisible: false})}
+                                        onItemSelected={() => this.setState({isMenuVisible: false})}
+                                        menuItems={this.createMenuItems(openPicker)}
+                                        anchorPosition={this.props.anchorPosition}
+                                    />
```

Comment:
@Beamanator I've removed the loading indicator here since the design doc mentions this:

> Both of these old flows (updating user & policy avatars) currently have loading states which will be removed since when a user updates an avatar (user or workspace) we can store that image data locally while we wait for the server to upload the file & respond with the avatar URL!

However, looking back, I think it might be good to keep the loading indicator since it'll potentially mitigate my concerns [here](https://github.com/Expensify/App/pull/10235#issuecomment-1228092978).

Here's how the loading indicator used to look. Do we want to keep it? This means I'll have to reintroduce a loading state into Onyx.
<img width="398" alt="Screen Shot 2022-08-26 at 2 36 22 PM" src="https://user-images.githubusercontent.com/31285285/186838111-03f13dd2-6680-4e43-84b5-6f251e3f5776.png">


cc @Expensify/design

Reply:
Interesting - personally I think it's ok not to have feedback because:
1. The image technically does update immediately, which is enough feedback for me that it succeeded ("optimistically")
2. If it fails we do show a nice error message (or if we go offline)

I'm definitely willing to discuss more but like you said @jasperhuangg maybe in a different issue or follow-up since the design doc does specifically call out having no feedback (pattern A)

Reply:
Cool cool! Just wanted to double-check. Let's wait and see what @Expensify/design has to say, but your logic makes sense to me and I'm more than happy to leave it like this.

Reply:
I agree with @Beamanator - the loading spinner kind of defeats the purpose of optimistic actions too right?